File: src/main/js/eweb/src/app/ehome/addemployee/employeeWorkPermitDetailsRequestDto.ts
export class EmployeeWorkPermitDetailsRequestDto{
endDate:string;
startDate:string;
validity:string;
workPermitNumber:string;

init(endDate:string,startDate:string,validity:string,workPermitNumber:string){
    this.endDate = endDate;
    this.startDate = startDate;
    this.validity = validity;
    this.workPermitNumber = workPermitNumber;
  }
}

File: src/main/java/com/employee/product/employeedetails/request/dto/EmployeeWorkPermitDetailsRequestDto.java
package com.employee.product.employeedetails.request.dto;

import java.sql.Date;

import lombok.Data;

@Data

public class EmployeeWorkPermitDetailsRequestDto {

	private String workPermitNumber;

	private Date startDate;

	private Date endDate;

	private int validity;
	
	private String documentName;
	
	private String documentType;
}
